pkg/gencmd/executor.go

```diff
@@ -0,0 +1,48 @@
+package gencmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type Executor interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Executor should have comment or be unexported